### PR TITLE
Issue #KN-245 feat: Knowlg player app integrate with player V1

### PR DESCRIPTION
**Pull request by gouravmore**
Issue #KN-245 feat: Knowlg player app integrate with player V1
Implementation Video:- https://drive.google.com/file/d/1VlWJtyNr06h2F6vvG8gB69XrdNxN9ByH/view?usp=sharing
APK:https://drive.google.com/file/d/1_VijW8kWIVplHn2E2JbTYlD-Mx01eCAc/view?usp=sharing

### CODE REVIEW

File: knowlg-app/proxy.config.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "/api/*": {
```
Comment (vaibhavbhuva) [inline knowlg-app/proxy.config.json]: Why you defined the separate proxy route for each path ?
 Why can't we use existing proxy config file here ?

Comment (gouravmore) [reply knowlg-app/proxy.config.json]: Created proxy config js file

File: projects/shared/src/lib/v1-player/v1-player.component.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      setTimeout(() => {
+        this.previewElement.nativeElement.contentWindow.initializePreview(this.playerConfig);
+        this.previewElement.nativeElement.contentWindow.addEventListener('message', resp => {
+          if (resp.data === 'renderer:question:submitscore') {
+            alert('Score has been submited succesfully');
+          } else if (resp.data && typeof resp.data === 'object') {
+            if (resp.data['player.pdf-renderer.error']) {
+              const pdfError = resp.data['player.pdf-renderer.error'];
+              if (pdfError.name === 'MissingPDFException') {
+                alert('This Pdf has some issue, please try with the differnet pdf content');
```
Comment (HarishGangula) [inline projects/shared/src/lib/v1-player/v1-player.component.ts]: Why alerts are used, use proper model to show error

File: projects/shared/src/lib/v1-player/v1-player.component.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+              }
+            } else if (resp.data && resp.data.event === 'renderer:maxLimitExceeded') {
+              alert('Max limit reached to attempt the quiz');
```
Comment (HarishGangula) [inline projects/shared/src/lib/v1-player/v1-player.component.ts]: Why alerts are used, use proper model to show error

File: projects/shared/src/lib/v1-player/v1-player.component.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            }
+          }
+        });
+      }, 100);
+    };
+  }
+
+  playerEventHandler(event) {
```
Comment (HarishGangula) [inline projects/shared/src/lib/v1-player/v1-player.component.ts]: playerEventHandler, telemetryEvent are used ?

File: projects/shared/src/lib/v1-player/v1-player.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { V1PlayerComponent } from './v1-player.component';
+
+describe('V1PlayerComponent', () => {
```
Comment (HarishGangula) [inline projects/shared/src/lib/v1-player/v1-player.component.spec.ts:5]: NO test cases

File: projects/shared/src/lib/v1-player/v1-player.component.html [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  <div class="relative9">
+      <!-- CC-Player Video -->
+      <div class="sb-single-resource player-maxwidth">
+      <div class="ui container">
+          <div class="cc-player sb-g">
+            <section class="sb-g-col-xs-12 sb-g-col-md-12 sb-g-col-lg-12 sb-g-col-xxxl-16 cc-player__video" id="player-area-height">
+              <div class="aspectratio sbt-shadow-radius" data-ratio="16:9" id="help-video-aspect-ratio" #aspectRatio>
+                <div class="h-100">
+                  <ion-content padding>
+                    <iframe #preview id="contentPlayer" style="width:100%;border:none;" title="Content Player"  src="content/preview/preview.html?webview=true" aria-label="Content Player" class="contentViewerIframeShadow" name="contentPlayer"></iframe>
```
Comment (HarishGangula) [inline projects/shared/src/lib/v1-player/v1-player.component.html]: No inline styles allowed, This very basic mistake

File: knowlg-app/src/app/home/services/content.service.spec.ts [hunk 7]
```diff
@@ -1,12 +1,15 @@
...
-
+import { HttpClientModule, HttpClient } from '@angular/common/http';
 import { ContentService } from './content.service';
 
 describe('ContentService', () => {
   let service: ContentService;
 
   beforeEach(() => {
-    TestBed.configureTestingModule({});
+    TestBed.configureTestingModule({
```
Comment (HarishGangula) [inline knowlg-app/src/app/home/services/content.service.spec.ts:10]: No test cases

**Review by HarishGangula — CHANGES_REQUESTED** [review]
No test cases are written

File: knowlg-app/src/app/home/services/content.service.spec.ts (hunk 7 above)
Comment (gouravmore) [reply knowlg-app/src/app/home/services/content.service.spec.ts:10]: Test cases are written v1 player integration changes

**Review by HarishGangula — APPROVED** [review]

**Review by vaibhavbhuva — APPROVED** [review]

**Review by vinukumar-vs — COMMENTED** [review]
What is changes in v2-player folder. I can see all are new files here.

**Pull request merged**